simulators/trin-bridge/src/main.rs

```diff
@@ -93,37 +144,35 @@ dyn_async! {
             Err(err) => panic!("{}", &err.to_string()),
         }
 
-        // start the bridge
-        let bridge = Bridge::new(
-            mode,
-            execution_api,
-            portal_clients,
-            header_oracle,
-            epoch_acc_path,
-        );
-        bridge.launch().await;
-
-        // wait 2 seconds for data to propagate
-        // This value is determined by how long the sleeps are in the bridge code
-        // So we may lower this or remove it depending on what we find.
-        tokio::time::sleep(Duration::from_secs(2)).await;
+        let client_a_enr = match client_a.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                panic!("Error getting node info: {err:?}");
+            }
+        };
+        client_a.test.start_client(TRIN_BRIDGE_CLIENT_TYPE.to_string(), Some(HashMap::from([(BOOTNODES_ENVIRONMENT_VARIABLE.to_string(), client_a_enr.to_base64()), (HIVE_CHECK_LIVE_PORT.to_string(), 0.to_string())]))).await;
+
+
 
         // With default node settings nodes should be storing all content
-        let values = std::fs::read_to_string("./test-data/test_data_collection_of_forks_blocks.yaml")
+        let values = std::fs::read_to_string(TEST_DATA_FILE_PATH)
             .expect("cannot find test asset");
         let values: Value = serde_yaml::from_str(&values).unwrap();
-        let comments = vec!["1 header", "1 block body", "100 header",
-            "100 block body", "7000000 header", "7000000 block body",
-            "7000000 receipt", "15600000 (post-merge) header", "15600000 (post-merge) block body", "15600000 (post-merge) receipt",
-            "17510000 (post-shanghai) header", "17510000 (post-shanghai) block body", "17510000 (post-shanghai) receipt"];
-
-        let mut result = vec![];
-        for (index, value) in values.as_sequence().unwrap().iter().enumerate() {
+        let content_vec: Vec<(HistoryContentKey, HistoryContentValue)> = values.as_sequence().unwrap().iter().map(|value| {
             let content_key: HistoryContentKey =
                 serde_yaml::from_value(value.get("content_key").unwrap().clone()).unwrap();
             let content_value: HistoryContentValue =
                 serde_yaml::from_value(value.get("content_value").unwrap().clone()).unwrap();
+            (content_key, content_value)
+        }).collect();
+        let comments = process_content(content_vec.clone());
 
+        // wait content_vec.len() seconds for data to propagate
+        // This value is determined by how long the sleeps are in the bridge code
+        tokio::time::sleep(Duration::from_secs(content_vec.len() as u64)).await;
```

Comment:
I think the second line of comments here is copy-pasta that can be removed:
```suggestion
        // wait content_vec.len() seconds for data to propagate, giving more time if more items are propagating
        tokio::time::sleep(Duration::from_secs(content_vec.len() as u64)).await;
```

If I'm wrong, then I definitely need more explanation to understand what it means here... :)